stingray/tests/test_crossspectrum.py

```diff
@@ -322,23 +375,23 @@ def test_norm_abs(self):
         assert len(cs.power) == 4999
         assert cs.norm == 'abs'
         abs_noise = 2. * self.rate1  # expected Poisson noise level
-        print(np.mean(cs.power), abs_noise)
-        assert np.isclose(np.mean(cs.power[1:]), abs_noise, atol=30)
+        assert np.isclose(np.mean(cs.power[1:]), abs_noise)
 
     def test_norm_leahy(self):
         with pytest.warns(UserWarning) as record:
             cs = Crossspectrum(lc1=self.lc1, lc2=self.lc1, norm='leahy')
         assert len(cs.power) == 4999
         assert cs.norm == 'leahy'
         leahy_noise = 2.0  # expected Poisson noise level
-        print(np.mean(cs.power), leahy_noise)
-        assert np.isclose(np.mean(cs.power[1:]), leahy_noise, atol=0.2)
+        assert np.isclose(np.mean(cs.power[1:]), leahy_noise, rtol=0.02)
 
     def test_norm_frac(self):
         with pytest.warns(UserWarning) as record:
```

Comment:
I don't remember what userwarning we were catching... I would recommend to test that `record` contains what we expect and not other things, at least once.